app/controllers/users_controller.rb

```diff
@@ -20,6 +21,6 @@ def create
   private
 
   def user_params
-    params.fetch(:user, {}).permit(:email, :password)
+    params.fetch(:user, {}).permit(:email, :password, :token)
```

Comment:
We shouldn't take the token from the user here. They give us email and password, we give them the user.